TS3Bot.py

```diff
@@ -806,25 +815,20 @@ def servergroupaddperm(sgid, permsid, permvalue):
                     for a in accs:
                         errored = False
                         try:
-                            u = User(ts3conn, unique_id = a, ex_hand = signal_exception_handler)
+                            u = User(ts3conn, unique_id=a, ex_hand=signal_exception_handler)
                             tsdbid = u.ts_db_id
-                            _, ex = ts3conn.ts3exec(lambda tsc: tsc.exec_("setclientchannelgroup"
-                                                                    , cid = cinfo.get("cid")
-                                                                    , cldbid = tsdbid
-                                                                    , cgid = contactgroup.get("cgid"))
-                                            , signal_exception_handler)
+                            _, ex = ts3conn.ts3exec(lambda tsc: tsc.exec_("setclientchannelgroup", cid=cinfo.get("cid"), cldbid=tsdbid, cgid=contactgroup.get("cgid")),
```

Comment:
This line appears to be 168 characters wide, which violates the new guideline of having `max_line_length = 120`, no?

Reply:
Limit for pythin files is 200.
120 seems to be a bit tight, and as everyone has widescreen dispalys nowadays, I choose 200.
Violations will be reported by the flake8 check